File: iniciante/1013/1013.py
# -*- coding: utf-8 -*-

entrada = input().split(' ')

a = int(entrada[0])
b = int(entrada[1])
c = int(entrada[2])

aux = (a + b + abs(a - b)) / 2
maior = (aux + c + abs(aux - c)) / 2

print(f'{maior:.0f} eh o maior')

File: iniciante/1013/1013.dart
import 'dart:io';

// RESPOSTA DO BEECROWD (SEM NULL-SAFETY)
/*
void main() {
  List entrada = stdin.readLineSync().split(" ");
  int a = int.parse(entrada[0]);
  int b = int.parse(entrada[1]);
  int c = int.parse(entrada[2]);

  double aux = (a + b + (a - b).abs()) / 2;
  double maior = (aux + c + (aux - c).abs()) / 2;

  print('${maior.toInt()} eh o maior');
}
*/

// RESPOSTA COM NULL-SAFETY
void main() {
  List entrada = stdin.readLineSync()!.split(" ");
  int a = int.parse(entrada[0]);
  int b = int.parse(entrada[1]);
  int c = int.parse(entrada[2]);

  double aux = (a + b + (a - b).abs()) / 2;
  double maior = (aux + c + (aux - c).abs()) / 2;

  print('${maior.toInt()} eh o maior');
}
